src/renderer/widgets/CreateWallet/model/create-multisig-wallet-model.ts

```diff
@@ -37,19 +38,28 @@ const $chain = createStore<ChainId | null>(null).reset(reset);
 const $signatories = createStore<Signatory[]>([]).reset(reset);
 const $error = createStore('').reset(reset);
 
+const $isEthereumChain = combine({ chainId: $chain, chains: networkModel.$chains }, ({ chainId, chains }) => {
+  return !!chainId && networkUtils.isEthereumBased(chains[chainId].options);
+});
+
```

Comment:
```suggestion
const $isEthereumChain = combine({
    chainId: $chain,
    chains: networkModel.$chains
  }, ({ chainId, chains }) => {
    return Boolean(chainId) && networkUtils.isEthereumBased(chains[chainId].options);
  });
```

Reply:
We should discuss and create formal rules for cases when we should use one line objects, and when not.

TypeScript doesn't recognise that `chainId` exists when use `Boolean` instead of `!!`

Reply:
Combines with first argument as object are multiline, combines with first argument not object are one-liners